pyat/at/lattice/lattice_object.py

```diff
@@ -274,19 +311,17 @@ def ohmi_envelope(self, *args, **kwargs):
         return ohmi_envelope(self, *args, **kwargs)
 
 
+Lattice.get_s_pos = get_s_pos
+Lattice.get_mcf = get_mcf
+Lattice.find_orbit4 = find_orbit4
+Lattice.find_sync_orbit = find_sync_orbit
+Lattice.find_orbit6 = find_orbit6
+Lattice.find_m44 = find_m44
+Lattice.find_m66 = find_m66
+
 if sys.version_info < (3, 0):
```

Comment:
I believe that if we move these to the bottom of `__init__` then we can do it the same way for all python versions:
```
def __init__():
    ...
    ...
    self.linopt.__func__.__doc__ += linopt.__doc__
    self.ohmi_envelope.__func__.__doc__ += ohmi_envelope.__doc__
```

Reply:
No, it does not work in Python 3
```
>       Lattice.linopt.__func__.__doc__ += linopt.__doc__
E       AttributeError: 'function' object has no attribute '__func__'

```

Reply:
I don't really understand what's happening, but it works…

Reply:
Ok, that makes sense.